examples/image-search/build.cpp

```diff
@@ -103,49 +107,52 @@ int main(int argc, char** argv) {
     std::vector<float> vec(vec_dim);
 
     // search for images in path and write embedding to database
-    for (const auto& base_dir : params.image_directories) {
+    for (const auto &base_dir : params.image_directories)
+    {
         fprintf(stdout, "%s: starting base dir scan of '%s'\n", __func__, base_dir.c_str());
-
-        for (auto const& dir_entry : std::filesystem::recursive_directory_iterator(base_dir)) {
-            if (!dir_entry.is_regular_file()) {
-                continue;
+        auto results = get_dir_keyed_files(base_dir, 0);
+
+        for (auto &entry : results)
+        {
+            printf("%s: processing %d files in %s\n", __func__, entry.second.size(), entry.first.c_str());
```

Comment:
```
main: starting base dir scan of '../tests/'
main: processing 2 files in
..
```
why does it say "processing x files in" and then only dots? code looks fine o.o

Reply:
verbose 2
```
main: starting base dir scan of '../tests/'
main: processing 2 files in
main: found image file '../tests//white.jpg'
main: found image file '../tests//red_apple.jpg'
```

Reply:
I just found printing every image file a little bit too verbose with default verbosity = 1 and used dots as an activity indicator, but maybe I can come up with something smarter.

Reply:
no that is fine. the problem is the missing string after `files in` `processing %d files in %s\n`